requirements/optional.txt

```diff
@@ -1,3 +1,4 @@
+albumentations
```

Comment:
I think we can be stricter here :) albumentation already comes with mmdetection and they use 
`albumentations>=0.3.2 --no-binary qudida,albumentations` [[ref.]](https://github.com/open-mmlab/mmdetection/blob/main/requirements/albu.txt)